File: static/integrations/admin-dashboard-dotnet-aspnet/Models/TrafficData.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace App.Models
{
    public class TrafficData
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public int Impressions { get; set; } 

        [Required]
        public int Clicks { get; set; }  

        [Required]
        public int Position { get; set; }

        [Required]
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}

File: static/integrations/admin-dashboard-php-laravel/app/Models/TrafficData.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class TrafficData extends Model
{
    use HasFactory;

    /**
     * The attributes that should be cast to native types.
     *
     * @var array
     */
    protected $casts = [
        'created_at' => 'datetime:Y-m-d',
    ];
}
